packages/patternlab/styleguide/source/_patterns/01-atoms/divider.twig

```diff
@@ -1 +1 @@
-<div class="ma__divider"></div>
+<hr class="ma__divider {{ (divider.thin == true)?"ma__divider--thin":'' }}" />
```

Comment:
Do you know why the markup is showing `<hr></hr>`
<img width="363" alt="Screen Shot 2021-11-02 at 1 33 15 PM" src="https://user-images.githubusercontent.com/5789411/139947823-59fbc3f5-e600-426d-9200-1a70f0940e1c.png">

This is actually invalid according to https://validator.w3.org/nu/#textarea
I feel like the thin divider shouldn't need this the :after sudo class either.

Reply:
<img width="867" alt="j-r-capture 2021-11-04 a la(s) 09 33 14" src="https://user-images.githubusercontent.com/141685/140314093-99213d59-bebf-45dc-beee-789587c19066.png">
That's something that the inspector is doing, if you check the actual source code, there isn't a separate closing tag.

Reply:
regarding the :after, it's needed for the padding on the container to have effect.

Reply:
interesting that the inspect is showing that